examples/hackernews/src/schema/base/schema.graphql

```diff
@@ -14,11 +14,11 @@ type Link {
   id: ID!
   description: String!
   url: String!
-  comments: [Comment!]!
+  comments: [Comment!]
```

Comment:
@Urigo What is the motivation for changing the type here? The field's resolver implementation will [always return an array and never `null`](https://www.prisma.io/docs/orm/reference/prisma-client-reference#findmany).

Reply:
that's true for Prisma, but I'm not sure it should be true for the GraphQL schema
We can choose to return what make sense for us

Reply:
I can change the resolver implementation, might make more sense for people
I changed this because we've got a couple of comments on this from users of the tutorial:
https://github.com/dotansimha/graphql-yoga/discussions/1167#discussioncomment-8652376
https://github.com/dotansimha/graphql-yoga/discussions/1167#discussioncomment-8987462

Reply:
@n1ru4l thanks for the conversation, I've added more context on this on the chapter and added the resolver implementation here:
https://github.com/dotansimha/graphql-yoga/pull/3524/commits/df4cc97984e7bffa16b8fa291c913f26aa35699e